docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst

```diff
@@ -1,8 +0,0 @@
-IMPOSTER NO MORE: How Tech Writers Can Shed Self-Doubt, Embrace Uncertainty, and Surf the Upcoming Swerve in Technical Documentation
```

Comment:
Hi @dwhitelf I'm talking about these 3 files which are unrelated to your PR. 

There is an error in the build system somewhere that deleted the files, and then 
I think you ran `git add -A` and which "added" the deleted files to the commit.

Reply:
I don't think I have the git skills to figure out how to get them back.

Reply:
No worries. I often revert to basics for this, and just copy the files back and commit them again.

Reply:
Sorted! Thanks